### PR TITLE
Merge `File` type stub back to source

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

File: github/File.py [hunk 1]
```diff
@@ -29,110 +29,76 @@
...
 
     @property
-    def status(self):
-        """
-        :type: string
-        """
+    def status(self) -> str:
         return self._status.value
 
-    def _initAttributes(self) -> None:
```
Comment (Borda) [inline github/File.py:123]: Why is this moved? let's keep the dif minimal for easier review...

Comment (trim21) [reply github/File.py:123]: EnricoMi think this is better

Comment (EnricoMi) [reply github/File.py:123]: @borda you are right, minimal diffs are better in general. In this particular case, having all attributes and types defined at the top of the class makes it more comprehensible, so while we are touching all files anyway, we are moving this method up.

**Review by EnricoMi — APPROVED** [review]
LGTM!
